src/block/Grass.php

```diff
@@ -91,14 +91,18 @@ public function onInteract(Item $item, int $face, Vector3 $clickVector, ?Player
 			TallGrassObject::growGrass($world, $this->position, new Random(mt_rand()), 8, 2);
 
 			return true;
-		}elseif($item instanceof Hoe){
+		}
+		if($face === Facing::DOWN){
```

Comment:
So it can be fertilized even from the bottom face? Seems weird

Reply:
Yes. I tested it in vanilla.

Reply:
To be honest I wouldn't be shocked if this was a bug in vanilla, but fine. I don't know if I like throwing an early return in like this though. Seems like it would be easy to get unexpected behaviour in the future if more actions need to be added.